projects/vdk-core/src/vdk/internal/plugin/plugin.py

```diff
@@ -50,6 +52,8 @@ def load_plugins_from_setuptools_entrypoints(self) -> None:
 
         try:
             self.__plugin_manager.load_setuptools_entrypoints(self.__group_name)
+        except ImportError as e:
+            exit_with_error(True, log, e)
```

Comment:
We exit directly on import error But   raise PluginException(message) on other type of errors. 

I don't think this kind of divergent logic is good.

Reply:
I have refactored both methods. The behaviour was the same, since when we raised the error, the pod would terminate - since no exception handling yet.